src/adc_readout/AdcTimeStamp.h

```diff
@@ -78,3 +98,73 @@ class TimeStamp {
   RawTimeStamp CTime{0, 0};
   ClockMode CMode{ClockMode::External};
 };
+
+template <typename ChronoTime>
+TimeStamp MakeExternalTimeStampFromClock(ChronoTime TimeNow) {
+  auto NowSeconds = std::chrono::duration_cast<std::chrono::seconds>(
+                        TimeNow.time_since_epoch())
+                        .count();
+  double NowSecFrac = (std::chrono::duration_cast<std::chrono::nanoseconds>(
+                           TimeNow.time_since_epoch())
+                           .count() /
+                       1e9) -
+                      NowSeconds;
```

Comment:
I am sure this is clang-format, but boy this break does nothing for the readability ;-)

Reply:
this is clang-format, and it happens frequently.